shell/client/accounts/account-settings.html

```diff
@@ -229,7 +229,7 @@ <h3 class="title-bar">Primary e-mail address for service-related notifications</
             </li>
           {{/with}}
           {{#if isPaymentsEnabled}}
-            {{> billingOptins}}
+            {{> billingOptins "" }}
```

Comment:
This broke the mailing list signup prompt, which needs `db` in order to display the user's current storage quota.

I've fixed this, but I'm worried that it's hard to know how many other things might be broken.